src/libraries/System.Security.Cryptography.Cose/src/System.Security.Cryptography.Cose.csproj

```diff
@@ -0,0 +1,36 @@
+<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <TargetFrameworks>$(NetCoreAppCurrent);$(NetCoreAppMinimum);netstandard2.0;$(NetFrameworkMinimum)</TargetFrameworks>
+    <Nullable>enable</Nullable>
+    <!-- TODO: Turn this into a nuget package -->
+    <!--<IsPackable>true</IsPackable>-->
+    <PackageDescription>TODO</PackageDescription>
+  </PropertyGroup>
```

Comment:
Something somewhere should be marking this library as preview-opt-in-required.  I expected that to be here-ish in the csproj.